client/related_items/helpers.js

```diff
@@ -39,4 +39,48 @@ const findDefaultStyle = (styles) => {
   });
 };
 
-module.exports = { getRelatedItems, getProductInfo, getProductStyles, findDefaultStyle };
\ No newline at end of file
+const defineMainProduct = (info, defaultStyle) => {
+  let price, thumbnailUrl, mainProduct;
+
+  !defaultStyle.sale_price
+    ? price = defaultStyle.original_price
+    : price = defaultStyle.sale_price;
+
+  !defaultStyle.photos[0].thumbnail_url
+    ? thumbnailUrl = 'https://cdn.shopify.com/s/files/1/0533/2089/files/placeholder-images-image_large.png?format=webp&v=1530129081'
+    : thumbnailUrl = defaultStyle.photos[0].thumbnail_url;
+
+  mainProduct = {
+    thumbnailUrl: thumbnailUrl,
+    category: info.category,
+    name: info.name,
+    price: price,
+    rating: '4.5',
```

Comment:
Are you hardcoding this in for now?

Reply:
Yes, good catch :) I want to re-use whatever Tatt has to calculate the rating so eventually this won't be hardcoded, probably updated via setState after an API call